fdpclient/operations.py

```diff
@@ -19,7 +19,7 @@ def create(url, data, format='turtle', **kwargs):
         **kwargs: Optional arguments that :func:`requests.request` takes.
     """
     logger.debug(f'Create metadata on {url} with the content: \n{data}')
-    if 'content-type' in kwargs:
+    if 'headers' in kwargs:
```

Comment:
So the idea here is that if the user specifies custom headers, then we set/overwrite `content-type`, and otherwise we leave it unspecified?

Reply:
That idea was based on our FDP implementation. Will change it to force using `turtle` if unspecified.